### DISCUSSION TITLE
Too much emphasis on development guidelines

### DISCUSSION THREAD

**Discussion by alan-amx** [tewhatuora/api-standards#23] [General]
### Summary

As raised previously, the text reads like a development guide, as opposed to a set of standards. I found it necessary to wade through large areas of informational text to find the meat of the applicable standard. The informational text was interesting but the risk is that many of the intended audience may tune out or skip most of the content, thereby missing the important standards information. I appreciate background is necessary to position a standard but in this case the balance doesn't seem quite right.

If the intent is to deliver a development guide and standards then I'd suggest pulling all the standards into their own section, where are they are easily digestible. Otherwise, remove the bulk of the how-to information to allow the standards themselves to take centre stage.

### Which area of the standards does this apply to?

- [X] Part A - API Concepts
- [X] Part B - API Security
- [X] Part C - API Design and Development
- [X] Part D - FHIR
- [ ] Community guidelines

**Comment by swithinfoote** [comment]
Thanks for your comments @alan-amx - we feel that this is more a discussion item than an issue thus will convert to a discussion to gather further opinion from standards participants/contributors.
Cheers.

**Comment by ChrisSquats** [comment]
Hi @alan-amx , Agree with you, In the next major release of the API standards we will add a consolidated list of standards so readers can view them all in a summarised place! Look forward to your feedback when it's released. I'll let you know when it's up in the draft section for early review.
